blockchain/ethereum/pancakeswap/wbnb deployer/0953af1d-ae69-47fe-aa66-085699649f88.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "blockchain/ethereum/pancakeswap/wbnb deployer",
```

Comment:
should have a more readable name. Eg Wraped BNB token deployer. The node name needs to start upper case.